### PR TITLE
Renews from master at 2017-05-23

### PR DESCRIPTION
**Pull request by nokitakaze**
Renews for #424.
- Main update is scrutinizer from PR #449

### CODE REVIEW

**Comment by ste93cry** [comment]
>I forgot about PHP 7.1 

This is not a commit message...

I would merge all the commit messages into only one and then squash everything. This way the history will look cleaner

**Comment by nokitakaze** [comment]
> squash everything

No. I did it on purpose. I want to keep entire log of commits

File: lib/Raven/Processor/SanitizeDataProcessor.php [hunk 1]
```diff
@@ -4,6 +4,7 @@
  * This file is part of Raven.
  *
  * (c) Sentry Team
+<<<<<<< HEAD:lib/Raven/Processor/SanitizeDataProcessor.php
```
Comment (nokitakaze) [inline lib/Raven/Processor/SanitizeDataProcessor.php]: Oooops

**Comment by nokitakaze** [comment]
Do I need to add PR #465 to this PR before merging?

**Comment by Jean85** [comment]
@nokitakaze if you want you can do it, merging the whole `improve_travis_build` branch.
Otherwise I will do it later.

**Comment by ste93cry** [comment]
>No. I did it on purpose. I want to keep entire log of commits. There was a lot of guys who did all this work. It's not about me or you. All commits should be keep in history of this project.

This is why I said to merge the messages from all commits into a single big description of a single big commit. This way you won't lose the history. But keeping merging thousands (and this is just a random number) between branches will only keep growing the history without any real benefit. I already know that those commits are on master, I don't need to have a copy of them also on `2.x`. Our current Git history is a damn mess

**Comment by nokitakaze** [comment]
There will be day when @dcramer does
```bash
git branch master
git merge 2.0 -Xrecursive theirs
```

I don\'t want to lose history at that day.

> Our current Git history is a damn mess

Being big project is so troublesome :3

**Pull request merged**